docs/content/documentation/systems/Kernels/phase_field/PFFracBulkRate.md

```diff
@@ -3,8 +3,25 @@
 # PFFracBulkRate
 !description /Kernels/PFFracBulkRate
 
+This kernel solves the second part of evolution function, in which the first part of the evolution function is solved with time derivative function.
+
+!parameters /Kernels/PFFracBulkRate
+{\bf beta} Auxiliary variable, which is the laplacian operator of c
+{\bf l} Length parameter, which determines the width of crack
+{\bf visco} the viscosity parameter, which reflects the transition right at crack stress
+{\bf c} Damage variable continuous between 0 and 1, 0 represents no damage, 1 represents fully cracked
```

Comment:
this list should be removed and the extended documentation strings merged into `PFFracBulkRate.C`